src/openforms/ui/static/ui/scss/components/skiplink/_skiplink.scss

```diff
@@ -1,7 +1,16 @@
 .skiplink {
   position: absolute;
-  top: -100vh;
-  left: -100vh;
+  padding: 6px;
+  background-color: var(--of-skiplink-bg, var(--of-button-primary-bg));
+  color: var(--of-skiplink-fg, var(--of-button-primary-fg));
+  border-bottom-right-radius: 6px;
+  border-right: 1px solid var(--of-skiplink-color-border, var(--of-button-primary-color-border));
+  border-bottom: 1px solid var(--of-skiplink-color-border, var(--of-button-primary-color-border));
+
+  &:not(&:focus) {
+    top: -100vh;
+    left: -100vh;
```

Comment:
typically you want to use `dvh` and `dvw`, since those account for the bars on mobile browsers (dynamic viewport height is mostly affected)